File: backend/src/main/java/pl/gddkia/security/EntityAuthUser.java
package pl.gddkia.security;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "USERS")
public class EntityAuthUser {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long id;
    String username;
    String password;
}

File: backend/src/main/java/pl/gddkia/security/EntityAuthUser.kt
package pl.gddkia.security

import jakarta.persistence.*

@Entity
@Table(name = "USERS")
data class EntityAuthUser(
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,
    var username: String? = null,
    var password: String? = null,
)
